types/api.go

```diff
@@ -425,11 +425,11 @@ type SettingsResponse struct {
 }
 
 type SignupRequest struct {
-	Email    string                 `json:"email,omitempty"`
-	Phone    string                 `json:"phone,omitempty"`
-	Password string                 `json:"password,omitempty"`
-	Data     map[string]interface{} `json:"data,omitempty"`
-
+	Email                   string                 `json:"email,omitempty"`
+	Phone                   string                 `json:"phone,omitempty"`
+	Password                string                 `json:"password,omitempty"`
+	Data                    map[string]interface{} `json:"data,omitempty"`
+	ConfirmationRedirectUrl string                 `json:"-"`
```

Comment:
I think `RedirectToURL` would be a preferable name here - more directly describes what the URL is for, and uses Go's preferred casing (i.e. `Url` → `URL`)